src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -105,17 +95,25 @@ public void execute(JobExecutionContext jobExecutionContext) {
     if (userAccountDetails != null) {
       String programmeName = jobDetails.getString(PROGRAMME_NAME_FIELD);
       LocalDate startDate = (LocalDate) jobDetails.get(START_DATE_FIELD);
-
+      NotificationType notificationType
+          = NotificationType.valueOf(jobDetails.get(NOTIFICATION_TYPE_FIELD).toString());
+      TisReferenceInfo tisReferenceInfo = new TisReferenceInfo(PROGRAMME_MEMBERSHIP,
+          jobDetails.get(TIS_ID_FIELD).toString());
       jobDetails.putIfAbsent("name", userAccountDetails.familyName());
 
-      log.info("Mock sent {} notification for {} ({}, starting {}) to {} using template {}", jobKey,
+      try {
+        emailService.sendMessage(personId, userAccountDetails.email(), notificationType,
+            templateVersion, jobDetails.getWrappedMap(), tisReferenceInfo, true);
+      } catch (MessagingException e) {
+        throw new RuntimeException(e);
```

Comment:
Here's the code smell. Because the overridden function doesn't throw a checked exception, the only obvious alternative would be annotating it with @SneakyThrows 🤔